run_tests

```diff
@@ -7,9 +7,9 @@ failed() {
 
 if [ -z "$1" -o "$1" == "unit" ]; then
     echo "=== test_unit ==="
-    nosetests3 --version || nose_missing=1
+    nosetests --version || nose_missing=1
```

Comment:
This will break running the testsuite on a normal system, where "python" and "nosetests" refer to python2, while python3 should be "python3" and "nosetests3".